jmin-jlink/README.md

```diff
@@ -0,0 +1,187 @@
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# jmin-jlink
+
+Includes:
+- **hellomodule.HelloWorld**, a simple app that contains classes, methods and fields - some of which should be removed during minimization;
+- **MinimimizeClassesPlugin**, a jlink plugin that minimizes an image using jarmin/jmin (jarmin/jmin-related code is commented out)
+- **MinimizeClassesPluginAgent**, a java agent that allows a custom implementation of a plugin to be used with jlink
+
+---
+
+## How to Run
+Follow these instructions to run jlink with the custom plugin (jlink option `--minimize-classes`) enabled by the java agent, 
+on a simple app contained in `hellomodule`. The resulting image is generated in the `build/image` directory and output from 
+the `jlink` command is piped to `minimizeclasses.out`.
+
+As this code involves Java modularity, **Java 11 or above is required**. 
+Take note of the path to your JDK `<PATH_TO_JDK>` as it is needed to run the below commands.
+
+From the `jmin-jlink` directory, run the following:
+
+### **Build and Run with `make`**
+If you have `make` installed (GNU Make 3.81 and above recommended), either set environment variable `JAVA_HOME` to the path to 
+your JDK, or append `JAVA_HOME="<PATH_TO_JDK>"` to the commands as indicated below.
+
+Then, you can simply run:
+- `make all`: Compile `hellomodule`, plugin, agent and create agent JAR\
+     - `make all JAVA_HOME="<PATH_TO_JDK>`: Pass JAVA_HOME explicitly
+- `make test`: Run the custom jlink plugin with hellomodule\
```

Comment:
I don't think you want the back-slashes at the ends of these lines.
Have a look at the [web view](https://github.com/sharon-wang/openj9-utils/tree/jmin-jlink/jmin-jlink) to see for yourself.